code/modules/mob/living/simple_animal/friendly/trader.dm

```diff
@@ -0,0 +1,242 @@
+/**
+ * # Trader
+ *
+ * A mob that has some dialogue options with radials, allows for selling items and buying em'
+ *
+ */
+
+/mob/living/simple_animal/hostile/retaliate/trader
+	name = "Trader"
+	desc = "Come buy some!"
+	icon = 'icons/mob/simple_human.dmi'
+	icon_state = "faceless"
+	maxHealth = 200
+	health = 200
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	attack_verb_continuous = "punches"
+	attack_verb_simple = "punch"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	del_on_death = TRUE
+	loot = list(/obj/effect/mob_spawn/human/corpse)
+	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+	unsuitable_atmos_damage = 2.5
+	casingtype = /obj/item/ammo_casing/shotgun/buckshot
+	wander = FALSE
+	ranged = TRUE
+	combat_mode = TRUE
+	move_resist = MOVE_FORCE_STRONG
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	sentience_type = SENTIENCE_HUMANOID
+	speed = 0
+	stat_attack = HARD_CRIT
+	robust_searching = TRUE
+	check_friendly_fire = TRUE
+	///Sound used when item sold/bought
+	var/sell_sound = 'sound/effects/cashregister.ogg'
+	///Phrases used when you talk to the NPC
+	var/list/lore = list("Hello! I am the test trader.",
+						"Oooooooo~!"
+						)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces

```suggestion
	var/list/lore = list(
		"Hello! I am the test trader.",
		"Oooooooo~!",
	)
```